openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -1246,3 +1246,11 @@ def _get_parameter_handler_class(self, tagname):
             msg += "Known parameter handler class tags are {}".format(self._parameter_handler_classes.keys())
             raise KeyError(msg)
         return ph_class
+
+    def __getitem__(self, val):
+        """Syntax sugar for lookikng up a ParameterHandler."""
+        if isinstance(val, str):
+            if val in self._parameter_handlers:
+                return self.get_parameter_handler(val)
+        elif isinstance(val, ParameterHandler):
```

Comment:
Ah, I was thinking that this could be the class/type itself, instead of an _instance_ of the class/type.

Reply:
I went ahead and covered both since I explicitly don't want to handle those cases now without more thinking. Let me know if that seems ok